src/main/java/uk/nhs/ctp/security/JWTLoginFilter.java

```diff
@@ -63,7 +63,8 @@ protected void successfulAuthentication(HttpServletRequest req, HttpServletRespo
 		UserEntity userEntity = userRepository.findByUsername(auth.getName());
 		List<CdssSupplier> cdssSuppliers;
 		if (userEntity.getRole().equals("ROLE_NHS") || userEntity.getRole().equals("ROLE_ADMIN")) {
-			cdssSuppliers = cdssSupplierRepository.findAll();
+			//TODO: CDSCT-139 (this code doesn't seem to be used as it gets overwritten straight away)
```

Comment:
Was it supposed to be an 'else'?

Reply:
I don't know, i'm not sure how this JWT auth on the EMS is supposed to behave.

Reply:
We need to do some more work on this - I've added CDSCT-166 to capture requirements around user accounts